source/components/triggering-changes-with-actions.md

```diff
@@ -156,3 +156,56 @@ a function instead.
 Actions in components allow you to
 decouple an event happening from how it's handled, leading to modular,
 more reusable components.
+
+## Calling Actions Across Component Layers
+
+When your components go multiple layers deep, its common to need to handle an action several layers up the tree.
+Using action helpers, its possible to make top level actions available at the bottom layers of your component tree
+without adding JavaScript code to the components in between.
+
+For example, we want to take account deletion out of the `user-profile` component and handle deletion in its parent.
+In our template in `user-profile.hbs`, we can change our action to call `deleteCurrentUser`,
+which will be defined on `system-preferences-editor`.
+
+```app/templates/components/user-profile.hbs
+{{button-with-confirmation onConfirm=(action deleteCurrentUser)
+  text="Click OK to delete your account."}}
+```
+
+Note that `deleteCurrentUser` is not in quotes as was the case [previously](#toc_passing-the-action-to-the-component)
+when the action was local to `user-profile`.
+This is because when you pass an action helper a string, it will attempt to invoke that function from the component's
+local `actions` object.
+
+When you pass an action helper a function reference as in this case, it will invoke the function from the component's
```

Comment:
Confusing wording. "When you pass a function reference to the `action` helper" maybe?